samples/quick-setup/README.md

```diff
@@ -1,4 +1,4 @@
-# Eventing RabbitMQ Broker with Trigger and DLQ Example
+# Eventing RabbitMQ Broker with Trigger/Source and DLQ Example
```

Comment:
I think this title can be better phrased and be without the `/`

Maybe just something generic like `Eventing RabbitMQ Examples`